### ISSUE TITLE
Warnings while using GetAll(Enchantments|Spells) with default values

### ISSUE THREAD

**Issue by Pop000100** [powerof3/PapyrusExtenderSSE#21]
`[08/22/2023 - 11:14:06PM] WARNING: Function PO3_SKSEFunctions.GetAllSpells(Keyword[] param1,bool param2) received NONE to non-object argument! Received types (none,bool) instead!
stack:
	[PopPAbilityFrameworkQuest (AE005901)].PopPFindAllCESpells.AddKeywordsToConstantEffects() - "-----------------------" Line 17
	[PopPAbilityFrameworkQuest (AE005901)].PopPFindAllCESpells.OnInit() - "-----------------------" Line 9`
	
Basically the same issue as #1
